browser/extensions/api/brave_rewards_api.cc

```diff
@@ -193,5 +197,21 @@ void BraveRewardsGetPendingContributionsTotalFunction::OnGetPendingTotal(
   Respond(OneArgument(std::make_unique<base::Value>(amount)));
 }
 
+BraveRewardsSaveAdsSettingFunction::~BraveRewardsSaveAdsSettingFunction() {
```

Comment:
It seems confusing to me that we're using `saveAdsSetting` to call `set_ads_enabled`. Why wouldn't we call this `setAdsEnabled`?

Reply:
@bridiver, I made this similar to the function in `brave_rewards_ui.cc`. Perhaps in the future one can do more management of Ads via the panel, then we have just this one method.